code/go/Makefile

```diff
@@ -23,4 +23,6 @@ check-spec:
 
 # Runs tests
 test:
-	@go test -v ./...
\ No newline at end of file
+    # -count=1 is included to bust the test cache. This way, if you run "make test" multiple times
+    # you will get fresh test results each time.
```

Comment:
nit: you can add here a note that changing source of mocked packages doesn't invalidate the cache.

Reply:
Done in d6af314.